centurio-restapi/src/main/kotlin/fr/hadaly/Main.kt

```diff
@@ -52,6 +52,9 @@ fun Application.module() {
         }
     }
 
+    install(CORS) {
+        anyHost()
```

Comment:
You should activate any host only on the dev environment, the prod environment will need the correct host.